Formula/kallisto.rb

```diff
@@ -0,0 +1,26 @@
+class Kallisto < Formula
+  desc "Quantify abundances of transcripts from RNA-Seq data"
+  homepage "https://pachterlab.github.io/kallisto/"
+  url "https://github.com/pachterlab/kallisto/archive/v0.43.1.tar.gz"
+  sha256 "2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e"
+
+  needs :cxx11
```

Comment:
`needs` goes after the `depends_on` series, on a line of its own, with one empty line above and below. (Don't ask me why 😁)